upb/def.c

```diff
@@ -1727,34 +1773,35 @@ static void create_enumdef(
 
   e->file = ctx->file;
   e->defaultval = 0;
+  e->value_count = 0;
+  e->values = symtab_alloc(ctx, sizeof(*e->values) * n);
 
   if (n == 0) {
     symtab_errf(ctx, "enums must contain at least one value (%s)",
                 e->full_name);
   }
 
   for (i = 0; i < n; i++) {
-    const google_protobuf_EnumValueDescriptorProto *value = values[i];
-    upb_strview name = google_protobuf_EnumValueDescriptorProto_name(value);
-    char *name2 = strviewdup(ctx, name);
-    int32_t num = google_protobuf_EnumValueDescriptorProto_number(value);
-    upb_value v = upb_value_int32(num);
+    const google_protobuf_EnumValueDescriptorProto *val_proto = values[i];
+    upb_enumvaldef *val = (upb_enumvaldef*)&e->values[e->value_count++];
```

Comment:
I think `e->value_count` is  `i` throughout this loop.  If so, I would find it slightly easier to read if you use i consistently and then outside the loop assign `e->value_count = n`

Reply:
Done.